### PR TITLE
feat: PostgreSQL destination plugin cloud config UI

### PR DESCRIPTION
**Pull request by jackcloudquery**
#### Summary

PostgreSQL destination plugin UI
Update scaffold with e2e tests

⚠️ **If you're contributing to a plugin please read this section of the [contribution guidelines](https://github.com/cloudquery/cloudquery/blob/main/CONTRIBUTING.md#open-core-vs-open-source) 🧑‍🎓 before submitting this PR** ⚠️

### CODE REVIEW

File: plugins/destination/postgresql/cloud-config-ui/src/form/index.tsx [hunk 1]
```diff
@@ -0,0 +1,263 @@
...
+              <TextField
+                error={!!fieldState.error}
+                fullWidth={true}
+                required={true}
+                helperText={
+                  fieldState.error?.message || (
+                    <>
+                      Specifies the update method to use when inserting rows.{' '}
+                      <a
+                        href="https://docs.cloudquery.io/docs/reference/destination-spec#write_mode"
```
Comment (cduflo) [inline plugins/destination/postgresql/cloud-config-ui/src/form/index.tsx:232]: does this work with the iframe embedded in the main app? Or do we need to use the messenger open_url here?

File: plugins/destination/postgresql/cloud-config-ui/src/form/radio.tsx [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+    disabled?: boolean;
+  }>;
+}
+
+/**
+ * This component displays a group of radio buttons.
+ *
+ * @public
+ */
+export function RadioGroupSelector<Value extends string | number | boolean>({
```
Comment (cduflo) [inline plugins/destination/postgresql/cloud-config-ui/src/form/radio.tsx:31]: we have ExclusiveToggle in the plugin-config-ui-lib that is pretty close to this, not sure if you want to swap it out?

**Review by cduflo — APPROVED** [review]
Nice! Nothing blocking, just 2 questions 👍

File: plugins/destination/postgresql/cloud-config-ui/src/form/index.tsx (hunk 1 above)
Comment (jackcloudquery) [reply plugins/destination/postgresql/cloud-config-ui/src/form/index.tsx]: Nice catch, thanks!

File: plugins/destination/postgresql/cloud-config-ui/src/form/radio.tsx (hunk 2 above)
Comment (jackcloudquery) [reply plugins/destination/postgresql/cloud-config-ui/src/form/radio.tsx]: Thanks, this is actually leftover

**Pull request merged**